### ISSUE TITLE
Current Hook UI is not working in CS-II v1.0.19f1

### ISSUE THREAD

**Issue by altafc22** [Captain-Of-Coit/hookui#29]
He devs, 
my CS-II updated to v1.0.19f1, and after that i'm not able to play the game, because HookUI is not compatible with new version

![image](https://github.com/Captain-Of-Coit/hookui/assets/25396264/f149f262-c0c0-4af9-a1b2-b6620d86dd6c)

**Comment by NurAminNRF** [comment]
Yes, the mod isn't compatible, but again, I sincerely appreciate the developers for cranking out such an amazing mod! Thanks for all the effort!

**Comment by defr0ke** [comment]
I think there will soon be an update to the mod that will be compatible with the new version.

The game can still be played. Simply remove the mod from the BepInEx (Unity Mono 6.0.0) folder

**Comment by Vaelek** [comment]
Any idea on when this might get some attention? I'm unsure of the differences but the Cities2Modding version of HookUI was updated the day it broke so it seemed to not be a large task. 
TIA.

**Comment by Xeventi** [comment]
Do you know when the mod will be updated?

**Comment by Vaelek** [comment]
> Do you know when the mod will be updated?

It may be worth checking the mods you're waiting on this for, all of the ones I was using have switched over to the Cities2Modding fork which was updated weeks ago.
